source/utilities.tex

```diff
@@ -11943,7 +11967,7 @@
 \begin{itemdescr}
 \pnum
 The maximum number of blocks that will be allocated at once
-from the upstream memory resource\iref{mem.res.monotonic.buffer}
+from the upstream memory resource\iref{mem.res.pool.overview}
```

Comment:
If you take my suggestion that the resource be permitted to allocate "one or more" chunks to replenish a pool, then this wording must be updated to replace the phrase "at once" with the phrase "in any single chunk".